projects/whydense/gsc/exps.cfg

```diff
@@ -628,52 +631,102 @@ dropout = 0.0
 log_interval = 400
 batches_in_epoch = 5121
 batch_size = 16
+
+
+# This one is like sparseCNN2WtSparsity18 but with a search for boost strength
+#
+[sparseCNN2WtBoostFactors2]
+cnn_out_channels = (64, 64)
+cnn_percent_on = (0.095, 0.125)
+cnn_weight_sparsity = (0.5, 0.2)
+linear_n = (1000,)
+linear_percent_on = (0.1,)
+weight_sparsity = (0.1,)
+k_inference_factor = 1.0
+iterations = 30
+boost_strength = tune.grid_search([0.0, 1.0, 1.5, 2.0])
+boost_strength_factor = tune.grid_search([0.9, 0.95, 0.97])
+learning_rate_factor = 0.9
+learning_rate = 0.01
+momentum = 0.0
+dropout = 0.0
+log_interval = 400
+batches_in_epoch = 5121
+batch_size = 16
 model_type = le_sparse
 activation_fct_before_max_pool = True
 
 
-[denseCNN2]
-linear_n = (1000,)
-linear_percent_on = (1.0,)
+# This one uses the best boost factors from sparseCNN2WtBoostFactors2
+# but with the best boost factors. Led to 12,300 noise accuracy
+[sparseCNN2WtSparsity22]
 cnn_out_channels = (64, 64)
-cnn_percent_on = (1.0, 1.0)
-cnn_weight_sparsity = (1.0, 1.0)
-k_inference_factor = 1.5
-iterations = 20
+cnn_percent_on = (0.095, 0.125)
+cnn_weight_sparsity = (0.5, 0.2)
+linear_n = (1000,)
+linear_percent_on = (0.1,)
+weight_sparsity = (0.1,)
+k_inference_factor = 1.0
+iterations = 30
 boost_strength = 1.5
-boost_strength_factor = 0.9
-learning_rate_factor = 0.8
+boost_strength_factor = 0.95
+learning_rate_factor = 0.9
 learning_rate = 0.01
-momentum = 0.9
-weight_sparsity = (1.0, )
-dropout = tune.grid_search([0.0, 0.1, 0.2])
-log_interval = 100
+momentum = 0.0
+dropout = 0.0
+log_interval = 400
 batches_in_epoch = 5121
-batch_size = 64
+batch_size = 16
 model_type = le_sparse
-use_batch_norm = True
+activation_fct_before_max_pool = True
+
 
 ; Larger sparse with lower weight sparsity
+; Potentially the next supersparse
 [SuperSparseCNN2]
-linear_n = (1500,)
-linear_percent_on = (0.067,)
 cnn_out_channels = (64, 64)
 cnn_percent_on = (0.095, 0.125)
-cnn_weight_sparsity = (1.0, 1.0)
-k_inference_factor = 1.5
-iterations = 25
+cnn_weight_sparsity = (0.5, 0.2)
+linear_n = (1500,)
+linear_percent_on = (0.1,)
+weight_sparsity = (0.05, )
+k_inference_factor = 1.0
+iterations = 30
 boost_strength = 1.5
 boost_strength_factor = 0.9
 learning_rate_factor = 0.9
 learning_rate = 0.01
 momentum = 0.0
-weight_sparsity = (0.1, )
 dropout = 0.0
 log_interval = 400
 batches_in_epoch = 5121
 batch_size = 16
 
 
+[denseCNN2]
+linear_n = (1000,)
+linear_percent_on = (1.0,)
+cnn_out_channels = (64, 64)
+cnn_percent_on = (1.0, 1.0)
+cnn_weight_sparsity = (1.0, 1.0)
+k_inference_factor = 1.5
```

Comment:
`k_inference_factor`  is not used. it should be removed

Reply:
I switched the default to 1.0, but kept the old experiments at 1.5